torchbox/torchbox.com
# Light mode - style form

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1551824181)

### Description of Changes Made

Styling of the mode switcher form


A few points to note:
- In order to fit the mode switcher in at smaller desktop sizes the breakpoint for switching to the desktop menu has been adjusted.
- I've adapted the previously unused `var(--color--accent-five)` to be used for the highlight light colour in figma, which is needed for the mode switcher
- There's some fiddly CSS to change the colour of the icon for the selected mode
- If JS is disabled, an extra button appears to submit the form and change the mode

### How to Test

Test on any page in your local build
You can see the updated styleguide at http://localhost:8000/pattern-library/
Try disabling JavaScript to see how it looks with the extra button in place

### Screenshots

<details>
  <summary>Expand to see more</summary>

Desktop light mode
![Screenshot 2024-07-10 at 10 17 24](https://github.com/torchbox/torchbox.com/assets/771869/8fae459b-ecaa-4417-8395-87c0425e8142)

Desktop dark mode
![Screenshot 2024-07-10 at 10 17 33](https://github.com/torchbox/torchbox.com/assets/771869/cf1096c2-1b58-47a6-9aaf-f1f0d022677e)

Mobile light mode
![Screenshot 2024-07-10 at 10 18 24](https://github.com/torchbox/torchbox.com/assets/771869/826330ae-b8d0-4ff3-90b6-f95a60f132a7)

Mobile dark mode
![Screenshot 2024-07-10 at 10 18 52](https://github.com/torchbox/torchbox.com/assets/771869/84af1735-7c25-4f7f-9762-07e5dc615f02)

Desktop with JS disabled
![Screenshot 2024-07-10 at 10 23 00](https://github.com/torchbox/torchbox.com/assets/771869/e8337568-e921-490b-8621-f230ebea9f5e)


</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
